### ISSUE TITLE
Style and Functionality improvements

### ISSUE THREAD

**Issue by Cdetviler** [aprsworld/craneWeb#11]
1. Change nav-bar to a different color

2. Change embedded pdf - bring in values from sql table instead - add download link

3. Try to get rid of show status button and fit everything on a tablet screen

**Comment by Cdetviler** [comment]
1. The navigation menu is now on one line no matter how many links there are. The background color is now the same color blue that is in our logo. I changed a number of other links to this blue color as well. The teal color that Ian made everything has very poor contrast with white text and makes it hard to read. This should improve that.

2. I scrapped the embedded PDF and created a big download link at the bottom of the certs page. I also created a query that grabs all of the relevant data from the calibration table and pulls it into a three column format.

3. I kept the status button for now. I tied everything to Javascript objects instead of html attributes. This adds a bit more stability and it is much faster than jQuery looking up the value attribute in the DOM. It's also a lot easier to debug. I could get rid of the status button entirely and just unhide the information if it is deemed unnecessary to have the button. I have been told that the reason this existed in the first place was so everything could fit on a tablet screen. To me, it seems to defeat the purpose when the information is hidden behind a button click. Perhaps I do not understand fully.

**Comment by aprsandrew** [comment]
The original reason for the "Show Status" button was to make a much less **cluttered** screen.  Although as a side effect, it did help it to fit nicely on a tablet screen.

We were going for simplified display that was similar to the CraneGate display, so a user in the field could bring it up on their tablet, laptop, smartphone, or whatever, and they'd have a screen similar to what the operator sees in-cab.  
All the extra status info doesn't really need to be displayed all the time, unless the user has a specific reason for wanting or needing to see that info.

**Comment by Cdetviler** [comment]
That makes sense. Well, unless Jim objects, I will leave the status button as is then. I think I corrected the weird unresponsive behavior behind the status button so it shouldn't be as irritating now. If it still appears as though the button is broken, let me know.

Also, I did not note above, but there is now a toggle button for the graph. Since their are most likely users who are used to the toggle already, I left the original functionality the same (i.e. clicking the battery and wind divs). I added tooltips and changed the mouse cursor to "pointer" to signify that they are indeed clickable items on the screen. The toggle button will allow users who did not know about the two different graphs, to now use them.
